src/js/gui.js

```diff
@@ -398,10 +398,20 @@ GuiControl.prototype.content_ready = function (callback) {
     this.switchery();
 
     if (CONFIGURATOR.connectionValid) {
+
+		//find page title
+        const pageTitle = $('div#content .tab_title');
+		var wikiLoc = pageTitle.attr("i18n");
```

Comment:
Please be careful with var. Nearly always you want to use const or let.
I think this should be const.

Reply:
Surely vat as it changes per page?